src/pf_types.h

```diff
@@ -2822,7 +2822,7 @@ struct pnet
          pf_lldp_name_of_device_mode_t mode;
       } name_of_device_mode;
       pf_port_t port[PNET_MAX_PORT];
-   } interface;
+   } dap;
```

Comment:
Why not prefix with pf_ or similar? Could even be pf_interface if that is more clear than pf_dap?

Reply:
Changed to "pf_interface".

Done